src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -64,11 +65,14 @@ public void lowerIntake() {
   public void periodic() {
     // This method will be called once per scheduler run
     if (runIntake && !reverseIntake) {
-      intakeSPrt.set(0.3);
-      intakeSPlf.set(0.3);
+      intakeSPrt.set(rollerSetpoint);
+      intakeSPlf.set(rollerSetpoint);
     } else if (runIntake && reverseIntake) {
-      intakeSPlf.set(-0.3);
-      intakeSPrt.set(-0.3);
+      intakeSPlf.set(-rollerSetpoint);
+      intakeSPrt.set(-rollerSetpoint);
+    } else {
+      intakeSPlf.set(0);
+      intakeSPrt.set(0);
```

Comment:
Good coding practice is to only set the motor values in one place.  Do all the math/selection first, then set the values once.

Example:
```
double setpoint = runIntake ? rollerSetpoint : 0;
if (reverseIntake)
  setpoint = -setpoint;

intakeSPlf.set(setpoint);
intakeSPrt.set(setpoint);
```